docs/getting-started.md

```diff
@@ -0,0 +1,3 @@
+# Getting Started
```

Comment:
Might duplicate `DEVELOPMENT.md`?

Reply:
Yup, this was the data-catalog equivalent of the external-credential-manager's `DEVELOPMENT.md`. Deleting this file.